test/cross-browser/report.js

```diff
@@ -29,14 +30,23 @@ var clearLogger = function() {
 };
 
 var logMessage = function(msg) {
-    return Q.fcall(function() {
-        var z = {
-            ts: Date.now(),
-            message: msg
-        };
-
-        return logger.push(z);
+    var timestampPromise = Q.fcall(function() {
+        return Date.now() + tsOffset;
     });
+
+    //if (typeof driver !== 'undefined') {
+    //    timestampPromise = driver.executeScript('return Date.now();');
+    //}
+
+    return timestampPromise
```

Comment:
It is invoked by the caller of this function, the `then` clause triggers the promise to evaluate itself.  

I will remove the commented out code, this was an early implementation of the timestamp sync.